src/AutorestSwift/Models/Complex/Parameter.swift

```diff
@@ -70,6 +70,36 @@ class Parameter: Value, CustomDebugStringConvertible {
         }
     }
 
+    var inPath: Bool {
+        if let x_in = extensions?["x-in"]?.value as? String,
+            x_in == ParameterLocation.path.rawValue {
+            return true
+        }
+
+        guard let location = paramLocation else { return false }
+        return location == ParameterLocation.path
+    }
+
+    var inQuery: Bool {
+        if let x_in = extensions?["x-in"]?.value as? String,
+            x_in == ParameterLocation.query.rawValue {
+            return true
+        }
```

Comment:
Is `x-in` just a duplicate way of specifying where a parameter should go? How does this differ from what we've been using?

Reply:
code is removed.